tests/unit/test_predictor.py

```diff
@@ -141,6 +142,16 @@ def test_csv_serializer_csv_reader():
         assert result == validation_data
 
 
+def test_csv_deserializer_array():
+    result = csv_deserializer(io.BytesIO(b'1,2,3'), 'text/csv')
+    assert result == [['1', '2', '3']]
+
+
+def test_csv_deserializer_2dimensional():
+    result = csv_deserializer(io.BytesIO(b'1,2,3\n3,4,5'), 'text/csv')
+    assert result == [['1', '2', '3'], ['3', '4', '5']]
+
+
```

Comment:
I would add a test with just one line and one value. YMMV

Reply:
done